docs/themes/scikit-learn-fork/javascript.html

```diff
@@ -0,0 +1,149 @@
+{% if theme_google_analytics|tobool %}
```

Comment:
I don't know enough JavaScript to review this properly, but I trust you!

Reply:
I don't know javascript either, I just copy/paste this from scikit-learn and anyway we don't have google analytics so it should not be a problem.